backend/game/engine/player_queue.py

```diff
@@ -0,0 +1,45 @@
+import itertools
+
+
+class GamePlayers():
+
+    def __init__(self):
+        self.all_players = []
+        self.dead_players = []
+        self.alive_players = []
+        self.current_player = None
+
+    def _add_initial_player(self, player):
+        self.all_players.append(player)
```

Comment:
You're appending whatever is given, but there really should be a player object being passed in. If it isn't a player object, an error should be raised.

Reply:
https://stackoverflow.com/a/32684829

and type annotations are not enforcable in python, do not want others to assume that python is acting as a statically typed language when they do use types
https://stackoverflow.com/a/32684754